wallet-new/src/Cardano/Wallet/API/V1/Migration.hs

```diff
@@ -1,14 +1,10 @@
 {- | This is a temporary module to help migration @V0@ datatypes into @V1@ datatypes.
 -}
-{-# LANGUAGE RankNTypes           #-}
-{-# LANGUAGE TypeSynonymInstances #-}
+{-# OPTIONS_GHC -Wno-unused-imports   #-}
```

Comment:
@adinapoli-iohk I didn't use autoexporter for this re-export module. Is this re-exporting module in a format you expect it to be?

Reply:
What about:

- Removing `-Wno-unused-imports`
- Removing `-Wno-dodgy-exports`
- Change 

```
import           Cardano.Wallet.API.V1.Migration.Helpers
import           Cardano.Wallet.API.V1.Migration.Types
```

into

```
import  qualified Cardano.Wallet.API.V1.Migration.Helpers as Exports
import qualified Cardano.Wallet.API.V1.Migration.Types as Exports
```

- Rename

```
 module Cardano.Wallet.API.V1.Migration.Helpers
, module Cardano.Wallet.API.V1.Migration.Types
 -- * Configuration re-exports
```

into 

```
module Exports
```

Would that still gives you the need for those 2 pragmas?

Reply:
@adinapoli-iohk I was able to remove those two pragmas without any further modifications (guess it is an artefact) 

```
module X (module Exports) where

import  qualified Cardano.Wallet.API.V1.Migration.Helpers as Exports
import qualified Cardano.Wallet.API.V1.Migration.Types as Exports
```
doesn't work as I am getting:
```
/home/akegalj/projects/serokell/cardano-sl/wallet-new/src/Cardano/Wallet/API/V1/Migration.hs:4:7: warning: [-Wdodgy-exports]
    The export item `module Exports' exports nothing
             
/home/akegalj/projects/serokell/cardano-sl/wallet-new/src/Cardano/Wallet/API/V1/Migration.hs:14:1: warning: [-Wunused-imports]
    The qualified import of ‘Cardano.Wallet.API.V1.Migration.Helpers’ is redundant
      except perhaps to import instances from ‘Cardano.Wallet.API.V1.Migration.Helpers’
    To import instances alone, use: import Cardano.Wallet.API.V1.Migration.Helpers()
             
/home/akegalj/projects/serokell/cardano-sl/wallet-new/src/Cardano/Wallet/API/V1/Migration.hs:15:1: warning: [-Wunused-imports]
    The qualified import of ‘Cardano.Wallet.API.V1.Migration.Types’ is redundant
      except perhaps to import instances from ‘Cardano.Wallet.API.V1.Migration.Types’
    To import instances alone, use: import Cardano.Wallet.API.V1.Migration.Types()
```
and later it breaks as it `doesn't export anything` so its missing stuff from those modules.

I believe this was possible to do somehow before.

@gromakovsky do you know how to re-export modules with a qualified names?

Reply:
AFAIK, for re-exports you should use unqualified imports.

>  I was able to remove those two pragmas without any further modifications (guess it is an artefact)

If there are no warnings without these pragmas, I think you there is nothing bad in using unqualified imports here for re-exports.

Reply:
> AFAIK, for re-exports you should use unqualified imports.

Dang, this is what happens when trying to type stuff on top of your head 😁

Reply:
Yes, that works . ty @adinapoli-iohk @gromakovsky